src/map/entities/mobentity.cpp

```diff
@@ -1046,9 +1046,214 @@ void CMobEntity::DropItems(CCharEntity* PChar)
+                        else
+                            AddItemToPool(3304, ++dropCount) // Shadow Geode
+                        break;
+                    default:
+                        break;
+                }
+            }
+            else if (xirand::GetRandomNumber(100) < 20)
+            {
+                uint8 dayRoll = xirand::GetRandomNumber(100);
+                switch (day)
+                {
+                    case 0: // Fire
+                        if (dayRoll < 45)
+                            AddItemToPool(3520, ++dropCount) // Ifritite
+                        else
+                            AddItemToPool(3297, ++dropCount) // Flame Geode
+                        break;
+                    case 1: // Earth
+                        if (dayRoll < 45)
+                            AddItemToPool(3523, ++dropCount) // Titanite
+                        else
+                            AddItemToPool(3300, ++dropCount) // Soil Geode
+                        break;
+                    case 2: // Water
+                        if (dayRoll < 45)
+                            AddItemToPool(3525, ++dropCount) // Leviatite
+                        else
+                            AddItemToPool(3302, ++dropCount) // Aqua Geode
+                        break;
+                    case 3: // Wind
+                        if (dayRoll < 45)
+                            AddItemToPool(3522, ++dropCount) // Garudite
+                        else
+                            AddItemToPool(3299, ++dropCount) // Breeze Geode
+                        break;
+                    case 4: // Ice
+                        if (dayRoll < 45)
+                            AddItemToPool(3521, ++dropCount) // Shivite
+                        else
+                            AddItemToPool(3298, ++dropCount) // Snow Geode
+                        break;
+                    case 5: // Thunder
+                        if (dayRoll < 45)
+                            AddItemToPool(3524, ++dropCount) // Ramuite
+                        else
+                            AddItemToPool(3301, ++dropCount) // Thunder Geode
+                        break;
+                    case 6: // Light
+                        if (dayRoll < 45)
+                            AddItemToPool(3526, ++dropCount) // Carbite
+                        else
+                            AddItemToPool(3303, ++dropCount) // Light Geode
+                        break;
+                    case 7: // Dark
+                        if (dayRoll < 45)
+                            AddItemToPool(3527, ++dropCount) // Fenrite
+                        else
+                            AddItemToPool(3304, ++dropCount) // Shadow Geode
+                        break;
+                }
+            }
+        }
+        // Geodes. Wiki's have conflicting info on mob lv required. One says 50 the other 75. Betting 50 is correct since the other tier is 80.
```

Comment:
I agree with this, most likely it is mob level 50. Maybe add a TODO to easy find this comment whenever someone wants to volunteer some time and confirm this? :smile_cat:

Reply:
> 
> 
> I agree with this, most likely it is mob level 50. Maybe add a TODO to easy find this comment whenever someone wants to volunteer some time and confirm this? 😸

Seems like I used to get them off the Funguar and Treants and not just the Hippogryphs and such that can spawn at 75 in Grauberg (S) when I was running trials there.  Was a good starting zone for several elemental weapons, with most targets being in the 60's.  IDK about 50 being the lower boundary, but could see it being 60 0r 65.

Reply:
> Seems like I used to get them off the Funguar and Treants and not just the Hippogryphs and such that can spawn at 75 in Grauberg (S) when I was running trials there. Was a good starting zone for several elemental weapons, with most targets being in the 60's. IDK about 50 being the lower boundary, but could see it being 60 0r 65.

@RAIST5150 my own personal theory is it will line up with the same lv mobs will be IEP to a lv99 when put to the test, which puts it lower than 75 but a little ways above the 50 ffxiclopedia claims. I recall getting some off exp mobs (IT's) while my character was on a job in the mid 50's back when they came out so I'm confident its at least lower than 75

Reply:
hehe... got lost in forums trying to see if I could spot something more concrete. Lowest levels I saw specifically cited fairly consistently were mobs that spawned around 60 (like a 58-62 range).  Some citings of mobs that could spawn lower, but some were not granting exp for them and they weren't sure of the levels that actually dropped the rocks, so it could actually be mobs that dip into the 50's.  Myself, I just remember targeting things that spawned around 60+ (like the Brasscaps and Knotty Treants I mentioned in Grauberg... 59-64 range).

It seemed to often beg the question of it basically being a mob that would grant exp to a 75 player.  IDK if that would be an acceptable rule of thumb to go with or not though.

Guess it is gonna be a hard one to figure out for certain at this point, with everyone being 99... forget how low the mobs can go and still grant EXP at 99.  Seems to be the only method one could use to figure out the mobs actual level?

Reply:
if looking at it from the player side there quite a range of levels mobs will still con EP at 75. iirc the exp tables for that correctly, that would put the lowest lv mob to give exp at lv 55 or 56. At 99 the lowest mob to potentially give exp is 59 due to the introduction of IEP, so pretty close ballpark.

I was figuring on leaving this a draft till I have time to make sure and then correct it, rather than leaving those comments from 5 years ago. I still have some jobs at lv 50 I can use for this on retail. I likely won't get aroudn to it till mid November or so tho. in the meantime I will just be making sure the logic still works after having updated 5yr old code.

Reply:
got confirmation it really is mob lv 50 or above. Also thought of a way to shorten the code by many lines, will update the branch when I have time and get this done.

Reply:
awesomeness great work! :D